Makefile

```diff
@@ -168,6 +168,22 @@ dep-ensure: ##@dependencies Dep ensure and apply all patches
 dep-install: ##@dependencies Install vendoring tool
 	go get -u github.com/golang/dep/cmd/dep
 
+update-geth: ##@dependencies Update geth (use GETH_BRANCH to optionally set the geth branch name)
+ifdef GETH_BRANCH
+	@# escape slashes
+	@GETH_BRANCH=$(echo $GETH_BRANCH | sed 's@\/@\\\/@g')
+	@# Update go-ethereum contraint branch
+	@sed -i 'N;N;s@\(\[\[constraint]]\n  name = "github.com\/ethereum\/go-ethereum"\n  branch =\)\(.*\)@\1 '"\"${GETH_BRANCH}\""'@g' Gopkg.toml
+endif
+	@dep ensure -v -update github.com/ethereum/go-ethereum
+	$(MAKE) dep-ensure
+	git add Gopkg.lock Gopkg.toml vendor/ _assets/patches/
```

Comment:
How can `_assets/patches/` change during this operation?

Reply:
I was also a little curious about this.. I added this as per step 4 of the [dependencies README](https://github.com/status-im/status-go/blob/develop/_assets/docs/DEPENDENCIES.md), but as far as I understand, it shouldn't.